server/plugin.go

```diff
@@ -480,6 +471,10 @@ func (p *Plugin) startMeeting(user *model.User, channel *model.Channel, meetingI
 			TemplateData: map[string]string{"Datetime": meetingLinkValidUntil.Format("Mon Jan 2 15:04:05 -0700 MST 2006")},
 		})
 	} else {
+		jitsiURL := strings.TrimSpace(p.getConfiguration().GetJitsiURL())
+		jitsiURL = strings.TrimRight(jitsiURL, "/")
+		meetingURL = jitsiURL + "/" + meetingID
+		meetingLink = meetingURL
```

Comment:
If `meetingLink` is always same as `meetingURL` then we can skip it right

Reply:
meetingLink is the url we are sending through the slack attachment .
Mainly the difference between meetingLink and meetingURL is meetingLink does not have jwt queryparam